dashboard/src/scenes/report/view.js

```diff
@@ -986,21 +979,19 @@ const ActionCompletedAt = ({ date, status, actions, setSortOrder, setSortBy, sor
   if (!data) return <div />;
 
   const moreThanOne = data.length > 1;
-
-  const completedAt = startOfToday().add(12, 'hours');
-
+  const dateForSelector = addHoursToDate(date, '12');
```

Comment:
pas besoin de cette abstraction
```suggestion
  const dateForSelector = dayjs(date).add(12, 'hour');
```